internal/core/src/index/ScalarIndex.h

```diff
@@ -50,6 +50,27 @@ class ScalarIndex : public IndexBase {
     virtual const TargetBitmap
     In(size_t n, const T* values) = 0;
 
+    virtual const TargetBitmap
+    InV2(size_t n,
```

Comment:
rename InApplyFilter or others, not use v1 v2 etc.

Reply:
OK.